iDiff/differs/pipDiff.go

```diff
@@ -86,11 +81,30 @@ func (d PipDiffer) getPackages(path string) (map[string]utils.PackageInfo, error
 					glog.Errorf("Could not find Python package %s for corresponding metadata info", packageName)
 					continue
 				}
+				currPackage := utils.PackageInfo{Version: version, Size: size}
+				addToMap(packages, packageName, pyVersion, currPackage)
+			}
 
-				packages[packageName] = utils.PackageInfo{Version: version, Size: size}
+			// if not package, check if Python file
```

Comment:
I see.  I adjusted the regexp a bit to handle packages with numbers or underscores in the name and for versions which don't fit the x.x.x standard.